src/encoded/static/components/app.js

```diff
@@ -235,23 +237,35 @@ var App = React.createClass({
             banners.push(<home.BannerLoader text='experiments' location='/search/?type=Experiment&award.project=4DN'/>);
             banners.push(<home.BannerLoader text='experiments' location='/search/?type=Experiment&award.project=External'/>);
             banners.push(<home.BannerLoader text='cell types' location='/search/?type=Biosource'/>);
+            // banners.push(<div> -- </div>);
+            // banners.push(<div> -- </div>);
+            // banners.push(<div> -- </div>);
             content = <home.HomePage banners={banners}/>;
             title = portal.portal_title;
         }else if (context) {
             var ContentView = globals.content_views.lookup(context, current_action);
-            content = <ContentView context={context} />;
-            title = context.title || context.name || context.accession || context['@id'];
-            if (title && title != 'Home') {
-                title = title + ' – ' + portal.portal_title;
-            } else {
-                title = portal.portal_title;
+            if (ContentView){
+                content = <ContentView context={context} />;
+                title = context.title || context.name || context.accession || context['@id'];
+                if (title && title != 'Home') {
+                    title = title + ' – ' + portal.portal_title;
+                } else {
+                    title = portal.portal_title;
+                }
+            }else{
+                // This code is here to deal with non-simultaneous changing
+                // of props.href and props.context. This content below should
+                // never be displayed.
+                content = <ErrorPage />;
+                title="Not Found";
```

Comment:
if the error is for non-simultaneous changing of props.href vs props.context should the tile / error page say that as opposed to Not Found?